templates/watchyourlan.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>WatchYourLAN</Name>
+  <Repository>aceberg/watchyourlan</Repository>
+  <Registry>https://hub.docker.com/r/aceberg/watchyourlan</Registry>
+  <Network>host</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/aceberg/WatchYourLAN</Support>
+  <Project>https://github.com/aceberg/WatchYourLAN</Project>
+  <Overview>Lightweight network IP scanner with web GUI.</Overview>
+  <Category>Tools: Network:Management</Category>
+  <WebUI>http://[IP]:[PORT:8840]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/watchyourlan.xml</TemplateURL/>
+  <Icon>https://raw.githubusercontent.com/aceberg/WatchYourLAN/main/assets/logo.png</Icon>
+  <Requires>Network Type needs to be set to Host.</Requires>
+  <Config Name="Config" Target="/data" Default="" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/wyl</Config>
+  <Config Name="Timezone" Target="TZ" Default="" Mode="" Description="" Type="Variable" Display="always" Required="false" Mask="false"></Config>
```

Comment:
Unraid automatically adds TZ to all containers, no need to have the user provide it.